app/views/users/_form.html.haml

```diff
@@ -5,6 +5,9 @@
       = f.text_field :spire, placeholder: '12345678@umass.edu'
       = f.form_group do
         = f.check_box :active
-      =f.form_group do
+      = f.form_group do
         = f.check_box :admin
+      = f.text_field :title, help: 'Will be shown on brochure', list: 'title-list'
+      %datalist#title-list
+        = options_for_select @titles
```

Comment:
I might suggest this to cut down the extra fluff in the controller, but I think this is another thing I don't have a strong opinion on in this case.
```suggestion
        = options_for_select User.select(:title).distinct.pluck(:title).compact
```

Reply:
Went back and forth on this, but I think you're right (as a side effect, also resolves a "don't use instance variables in partials" Rubocop offense).